getting_started/cloud_infrastructure_access/dedicated-aws-peering.adoc

```diff
@@ -0,0 +1,123 @@
+[id="dedicated-aws-peering"]
+= AWS VPC Peering Configuration
+include::modules/common-attributes.adoc[]
+:context: dedicated-aws-peering
+toc::[]
+
+This is the sample process for configuring an AWS VPC containing an OpenShift
+Dedicated cluster to peer with another AWS VPC network.  For more information
+about creating an AWS VPC Peering connection or for other possible
+configurations, see the
+link:http://docs.aws.amazon.com/AmazonVPC/latest/PeeringGuide/Welcome.html[AWS Peering Guide].
+
+[id="dedicated-vpc-peering"]
+== VPC Peering
+This section details the steps required to set up a VPC peering connection between two VPCs on two separate AWS accounts. To keep things clear, these are the terms that will be used for the rest of this explanation:
+
+* *{product-title} AWS Account* - The AWS account that contains the OpenShift Dedicated cluster.
+* *{product-title} Cluster VPC* - The VPC that contains the OpenShift Dedicated.
+* *Customer AWS Account* - Your non-{product-title} AWS account that you would like to peer with.
+* *Customer VPC* - The VPC in your AWS Account that you would like to peer with.
+* *Customer VPC Region* - The region that the customer’s VPC resides.
+
+[NOTE]
+====
+As of July 2018, AWS supports inter-region VPC peering between all commercial regions link:https://aws.amazon.com/vpc/faqs/#Peering_Connections[excluding China].
+====
+
+[id="initiate-accept-peering"]
+== Initiate and Accept the VPC Peer Request
+
+.Prerequisites
+
+* Gather some information about the Customer VPC that you’ll need to initiate the peering request - Customer AWS account number, Customer VPC ID, Customer VPC Region, Customer VPC CIDR
+* Check the CIDR block used by the {product-title} Cluster VPC. If it overlaps or matches the CIDR block for the Customer VPC, then peering between these two VPCs is link:https://docs.aws.amazon.com/vpc/latest/peering/invalid-peering-configurations.html[not possible]. If the CIDR blocks do not overlap, proceed to step 3.
+
+.Procedure - Initiate the VPC Peering Request
+The first step in this process is to send a VPC peering connection request from the OpenShift Dedicated AWS Account to the Customer AWS Account.
+
+. xref:../../getting_started/cloud_infrastructure_access/dedicated-aws-access.html#dedicated-aws-ocm-iam-role[Log into the Web Console for the {product-title} AWS Account] and navigate to the *VPC Dashboard* in the region where the cluster is being hosted.
+. Go to the *Peering Connections* page and click the *Create Peering Connection* button.
+. Double check details of the account you are logged in to and double-check the details of the account/VPC you are connecting to (from the first step above).
+.. Peering connection name tag - A descriptive name for the VPC Peering Connection.
+.. VPC (Requester) - This should be the {product-title} Cluster VPC ID, select from the dropdown.
```

Comment:
From ISG: "Refer to a drop-down list by its label, followed by _list_." If these GUI lists have a label, use that, otherwise "drop-down list".